### DISCUSSION TITLE
Nonprofit status plans: c6 or c3?

### DISCUSSION THREAD

**Discussion by ShaneCurcuru** [commonhaus/foundation#125] [Q&A] [answered]
Welcome new foundation on the block!  I see legally you've been around (under a previous name) as a c6, I was wondering if there were any plans to switch to a c3 organization ever, or if your choice of c6 is intentional?

Relatedly, I'd strongly suggest adding a prominent "Projects" feature page at the top of your nav structure: featuring the project communities you host is a key way to get broader interest both in the foundation as well as your projects.

In any case, good luck, nice to see such an organized setup and a new foundation focused on helping maintain communities. 😻

**Comment by ebullient** [comment, marked as answer]
>  I see legally you've been around (under a previous name) as a c6, I was wondering if there were any plans to switch to a c3 organization ever, or if your choice of c6 is intentional?

As you've mentioned, the c6 status is somewhat historical. There are trade-offs between the two statuses. At the moment, we're focused on the transition, getting our feet under us, and going through the remaining details to get the engine working well. The c6 vs c3 question is one that could be revisited later.

>.. I'd strongly suggest adding a prominent "Projects" feature page at the top of your nav structure

I shall take that suggestion. Thank you for making it!

**Reply by ShaneCurcuru** [reply]
That all makes perfect sense, and I also see after scrolling you have your list of projects **right there** at the bottom of the page... _(duh, Shane!)_ but a topline nav entry is still a good idea.
